sites/partners/src/applications/PaperApplicationDetails/sections/DetailsMultiselectQuestions.tsx

```diff
@@ -1,54 +1,47 @@
-import React, { useContext, useMemo } from "react"
+import React, { useContext } from "react"
 import { t, GridSection, ViewItem, GridCell } from "@bloom-housing/ui-components"
+import { listingSectionQuestions } from "@bloom-housing/shared-helpers"
 import { ApplicationContext } from "../../ApplicationContext"
-import { InputType, AddressCreate } from "@bloom-housing/backend-core/types"
+import { InputType, AddressCreate, ApplicationSection } from "@bloom-housing/backend-core/types"
 import { DetailsAddressColumns, AddressColsType } from "../DetailsAddressColumns"
 import { useSingleListingData } from "../../../../lib/hooks"
 
-type DetailsPreferencesProps = {
+type DetailsMultiselectQuestionsProps = {
   listingId: string
+  applicationSection: ApplicationSection
+  title: string
 }
 
-const DetailsPreferences = ({ listingId }: DetailsPreferencesProps) => {
+const DetailsMultiselectQuestions = ({
+  title,
+  applicationSection,
+  listingId,
+}: DetailsMultiselectQuestionsProps) => {
   const { listingDto } = useSingleListingData(listingId)
 
   const application = useContext(ApplicationContext)
 
-  const listingPreferences = listingDto?.listingPreferences
-  const preferences = application?.preferences
+  const listingQuestions = listingSectionQuestions(listingDto, applicationSection)
 
-  const hasMetaData = useMemo(() => {
-    return !!listingPreferences?.filter(
-      (listingPreference) => listingPreference.preference?.formMetadata
-    )?.length
-  }, [listingPreferences])
-
-  if (!hasMetaData) {
-    return null
+  if (listingQuestions?.length === 0) {
+    return <></>
   }
 
-  return (
-    <GridSection
-      className="bg-primary-lighter"
-      title={t("application.details.preferences")}
-      inset
-      columns={2}
-    >
-      {listingPreferences?.map((listingPreference) => {
-        const metaKey = listingPreference?.preference?.formMetadata?.key
-        const optionDetails = preferences.find((item) => item.key === metaKey)
+  const questions = application[applicationSection]
 
+  return (
+    <GridSection className="bg-primary-lighter" title={title} inset columns={2}>
+      {listingQuestions?.map((listingQuestion) => {
         return (
-          <GridCell key={listingPreference.preference.id}>
-            <ViewItem
-              label={t(`application.preferences.${metaKey}.title`, {
-                county: listingDto?.countyCode,
-              })}
-            >
+          <GridCell key={listingQuestion?.multiselectQuestion.text}>
+            <ViewItem label={listingQuestion?.multiselectQuestion.text}>
               {(() => {
-                if (!optionDetails?.claimed) return t("t.none")
+                const appQuestion = questions?.filter(
+                  (question) => question.key === listingQuestion?.multiselectQuestion.text
+                )[0]
```

Comment:
Since you are just getting the first one that is returned you can do 
`questions?.find` instead